docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +106,255 @@ To manage file storages:
+   console.log(fileContents);
+   // FileReader is capable of extracting blob contents
+   var reader = new FileReader();
+   // Event fires after the blob has been read/loaded
+   reader.addEventListener('loadend', function(contents)
+   {
+      const text = contents.srcElement.result;
+      console.log('File contents: ' + text);
+   });
+   // Start reading the blob as text
+   reader.readAsText(fileContentsInBlob);
+   ```
 
+   - when file is no longer needed, remember to close it
    ```
-           if (files[i].isDirectory == false) {
-               files[i].readAsText(function(str) {
-                   console.log('File content: ' + str);
-               }, null, 'UTF-8');
-           }
-       }
-   }
+   fileHandleRead.close();
    ```
 
+> **Note**
+> When you use readString(), readData(), readBlob() or one of their NonBlocking equivalent, FileHandle position is set right  where the reading operation ended, to read file again from the beggigning, use seek() method on Filehandle object.
+
 ## Managing Files and Directories
 
 You can manage files and directories in many ways:
 
-- You can read and write to a file by first using the `openStream()` method to open the file. You can specify the file mode and encoding.The `openStream()` method returns a `FileStream` object (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#FileStream), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#FileStream), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#FileStream) applications), which is a handle to the opened file. All actual operations, such as read, write, or close, on the file are performed through the `FileStream` object based on a position attribute, which represents the current position in the file.
- - You can copy and move files and directories within the virtual file system with the `copyTo()` and `moveTo()` methods.If a file or directory of the same name already exists in the target location, the overwrite input parameter of the method defines whether the existing file is overwritten.
-	> **Note**
-	> The file or directory to be copied or moved must be located under the current directory.
+- You can read and write to a file by first using the `openFile()` method to open the file. You can specify the file mode.The `openFile()` method returns a `FileHandle` object (in [mobile](../../api/latest/device_api/mobile/tizen/filesystem.html#FileHandle), [wearable](../../api/latest/device_api/wearable/tizen/filesystem.html#FileHandle), and [TV](../../api/latest/device_api/tv/tizen/filesystem.html#FileHandle) applications), which is a handle to the opened file. All actual operations, such as read, write, or close, on the file are performed through the `FileHandle` object based on a curent position, which can be obtained or modified through calling `seek()` operation on `FileHandle` object.
```

Comment:
You can specify the file mode.<missing space here>The `openFile()` method return

such as read, write <not needed coma here > or close,